### PR TITLE
Add Recipe: Few-shot Finetuning and Evaluation

### PR DESCRIPTION
**Pull request by fayvor**
Implements [pm/#109](https://github.com/ibm-granite-community/pm/issues/109):
* Configure a Model and Trainer for fine-tuning
* Train and Evaluate the Model
* Plot Predicted vs Actual for a sample of time intervals

### CODE REVIEW

**Comment by fayvor** [comment]
@wgifford please check out this recipe after you look at the Preprocessing one.

File: recipes/Time_Series/Few-shot_Finetuning_and_Evaluation.ipynb [hunk 1]
```diff
@@ -0,0 +1,891 @@
...
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Train the Model\n",
+    "\n",
+    "This takes about 2 minutes."
```
Comment (fayvor) [inline recipes/Time_Series/Few-shot_Finetuning_and_Evaluation.ipynb:644]: @wgifford could you provide a description of what is happening under the hood with fine-tuning? Is this creating an adapter? Or is it coming up with a few-shot prompt?

**Comment by fayvor** [comment]
@wgifford please review.

File: recipes/Time_Series/Few-shot_Finetuning_and_Evaluation.ipynb [hunk 2]
```diff
@@ -0,0 +1,875 @@
...
+    "  num_input_channels=len(target_columns), # Number of input columns.\n",
+    "  head_dropout=0.7,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Freeze the TTM Backbone"
```
Comment (fayvor) [inline recipes/Time_Series/Few-shot_Finetuning_and_Evaluation.ipynb:534]: @wgifford what is happening in this step, and why do we do it? Could you point me to some documentation or provide a description?

**Comment by fayvor** [comment]
@wgifford is this ready for review?

**Comment by wgifford** [comment]
I am still waiting on some info to make changes.

**Review by wgifford — APPROVED** [review]

**Pull request merged**